src/pages/QueueStatus/Components/Footer.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import styled from "styled-components";
+
+import Box from "@material-ui/core/Box";
+import { Label } from "./Label";
+
+import FooterIcon from "@src/assets/icons/footer.svg";
+import googleIcon from "@src/assets/icons/google-icon.svg";
+import NOSIcon from "@src/assets/icons/nos-icon.svg";
+import Tech4CovidIcon from "@src/assets/icons/Logo-Tech4COVID19-white.svg";
+
+import { useTranslation } from "react-i18next";
+
+const CustomizedFooter = styled.div`
```

Comment:
We are repeating the property flex-direction. Thanks to the cascading they are overwritting the `column` value but it would be better to have it just once:
```
flex-direction: ${({ isDesktop }) => isDesktop ? "flex-end" : "row-reverse"};
```